ApplicationInsights/JavaScriptSDK.Interfaces/IPageViewTelemetry.ts

```diff
@@ -3,32 +3,42 @@
  */
 export interface IPageViewTelemetry {
     /*
-     * name The string you used as the name in startTrackPage. Defaults to the document title.
+     * name String - The string you used as the name in startTrackPage. Defaults to the document title.
      */
-    name: string;
-    
+    name?: string;
+
     /*
-     * url  String - a relative or absolute URL that identifies the page or other item. Defaults to the window location.
-     */ 
-    url: string;
+     * uri  String - a relative or absolute URL that identifies the page or other item. Defaults to the window location.
+     */
+    uri?: string;
 
     /*
-     * id  String - unique identifier for the page view
-     */ 
-    id: string;
+     * id string - unique page identifier for the page view
+     */
+    id?: string;
```

Comment:
so sorry, but this is also not exposed to customer, we set it up internally like ver.